### ISSUE TITLE
Breaking in VSCode

### ISSUE THREAD

**Issue by jpike88** [palantir/tslint#4898]
### Bug Report

- __TSLint version__: 5.20.1
- __TypeScript version__: 3.6.4, (also on 3.7.2)
- __Running TSLint via__: VSCode

#### Reproduction using [TSLint Playground](https://palantir.github.io/tslint-playground)

Looks like tslint just fails on VSCode, using TypeScript 3.6.3 causes it to reactivate.

#### TypeScript code being linted

```ts
var wat;
```

^ my rules should be showing a warning on this code. should be let or const.

#### Actual behavior

Doesn't do anything

#### Expected behavior

Should work with latest TS

Doesn't look like many things were added in 3.6.4
https://github.com/microsoft/TypeScript/milestone/104?closed=1

**Comment by jpike88** [comment]
I think I'm going insane.

It looks like a general failure.

![Screen Shot 2019-12-03 at 11 57 01 am](https://user-images.githubusercontent.com/9585787/70021663-07642b80-15c4-11ea-8877-22a20f49889c.png)

Workspace version actually fails across the board, but using VSCode's version works. Wat

**Comment by jpike88** [comment]
false positive, actually vscode broke it

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
